### PR TITLE
Goals page

### PR DESCRIPTION
**Pull request by Doroshkeviches**

### CODE REVIEW

File: frontend/src/Pages/goalsPage/scripts/setGoalsToLocalStorage.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { arrayOfGoals } from './arrayOfGoals';
+export function setGoalsToLocalStorage() {
+  arrayOfGoals.length = 0;
```
Comment (RetiredHedgehog) [inline frontend/src/Pages/goalsPage/scripts/setGoalsToLocalStorage.ts:3]: What the point of creating an empty array in a separate file, importing it here and immediately setting its length to zero? Get rid of `arrayOfGoals`  and use `const arrayOfGoals = [];` inside of the function?

Comment (RetiredHedgehog) [reply frontend/src/Pages/goalsPage/scripts/setGoalsToLocalStorage.ts:3]: And if you are planning to use `arrayOfGoals` as a global variable that you can import at any point, please don't. Instead pass it as a function argument.

Comment (RetiredHedgehog) [reply frontend/src/Pages/goalsPage/scripts/setGoalsToLocalStorage.ts:3]: It also would be better to split this function into two: `setGoalsToLocalStorage` which will expect an array and set it to local storage and `getGoalsFromDOM` (or something better) that will get values from `labelText` and return them.

**Review by RetiredHedgehog — APPROVED** [review]

File: frontend/src/Pages/goalsPage/scripts/setGoalsToLocalStorage.ts (hunk 1 above)
Comment (Doroshkeviches) [reply frontend/src/Pages/goalsPage/scripts/setGoalsToLocalStorage.ts:3]: if I don't clean the array every time the function is called, then my targets will be duplicated and clogged localStrorage

**Review by RetiredHedgehog — APPROVED** [review]

File: frontend/src/Pages/goalsPage/style/style.sass [hunk 2]
```diff
@@ -0,0 +1,65 @@
+.myGoals
+    font-size: 36px
```
Comment (inanimate631) [inline frontend/src/Pages/goalsPage/style/style.sass:2]: constans

File: frontend/src/Pages/goalsPage/style/style.sass [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    margin-right: 16px
+    fill: $color-gren
+    cursor: pointer
+.popupActive
+    position: absolute
+    background: rgba(0, 0, 0, 0.4)
+    height: 100vh
+    width: 100%
+    top: 0px
+    left: 0px
```
Comment (inanimate631) [inline frontend/src/Pages/goalsPage/style/style.sass:65]: new line

File: frontend/src/Routes/app.routes.ts [hunk 4]
```diff
@@ -20,4 +21,5 @@ export const appRoutes: routesObj[] = [
   },
   { path: '', component: autorizationTemplate },
   { path: 'Calculator', component: nutritionCalculator.template },
+  { path: 'Goals', component: goalPageTemplate },
```
Comment (inanimate631) [inline frontend/src/Routes/app.routes.ts:24]: i sing you need pull and merge dev to avoid conflicts

**Review by inanimate631 — APPROVED** [review]

**Pull request merged**